deployment/base/rbac/master-clusterrole.yaml

```diff
@@ -11,3 +11,18 @@ rules:
   - get
   - patch
   - update
+  - list
+- apiGroups:
+  - topology.node.k8s.io
+  resources:
+  - noderesourcetopologies
+  verbs:
+  - create
+  - get
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
```

Comment:
Sure, I will a create service account and rbac rules for the topologyupdater when it is deployed as a separate pod but when topology-updater and master are deployed as separate containers in a single pod because a serviceaccount is defined per pod we would need to have a service account that allows access to pod. Let me know if that is okay?

Reply:
> but when topology-updater and master are deployed as separate containers in a single pod because a serviceaccount is defined per pod

Hmm, true, I didn't think about that.

> Let me know if that is okay?

Yes, if you think it's ok. You could probably just patch the clusterrole in this case(?) There's no way around this when all daemons are running in the same pod 🤓

Reply:
I was thinking that I will define a separate cluster role with just this capability when nfd-topology-updater is in a separate pod where nfd-master doesn't need this capability (and uses cluster-role without this) and only for the cases when nfd-topology-updater and nfd-master are in a single pod we use a combined cluster role. It is working nicely, will post the fix soon (working through all the review comments and corresponding fixes).